ftplugin/javascript/cfi.vim

```diff
@@ -0,0 +1,181 @@
+scriptencoding utf-8
+
+" ============================================================================
+" Bootstrap
+" ============================================================================
+
+if get(g:, 'cfi_disable') || get(g:, 'loaded_cfi_ftplugin_javascript')
+    finish
+endif
+let g:loaded_cfi_ftplugin_javascript = 1
+
+" Saving 'cpoptions' {{{
+let s:save_cpo = &cpo
+set cpo&vim
+" }}}
+
+" ============================================================================
+" Options
+" ============================================================================
+
+if !exists('g:cfi_javascript_show_params')
```

Comment:
However, I'm from mobile now.
I review this PR in details when I get home.
Thanks for PR! :)